templates/README.example.md

```diff
@@ -1,62 +1,48 @@
-[![Dependency Status](https://gemnasium.com/ADD_GEMNASIUM_TOKEN.svg)](https://gemnasium.com/renuo/<appname>) [![Build Status](https://travis-ci.com/renuo/<appname>.svg?token=ADD_TRAVIS_TOKEN&branch=master)](https://travis-ci.com/renuo/<appname>) [![Build Status](https://travis-ci.com/renuo/<appname>.svg?token=ADD_TRAVIS_TOKEN&branch=develop)](https://travis-ci.com/renuo/<appname>) [![Build Status](https://travis-ci.com/renuo/<appname>.svg?token=ADD_TRAVIS_TOKEN&branch=testing)](https://travis-ci.com/renuo/<appname>)
+# Project Title
+Short project description
 
-# <appname>
+## Environments
 
-## Important Links
+| Branch  | Domain                          | Deployment | CI                                      |
+| ------- | ------------------------------- | ---------- | --------------------------------------- |
+| develop | https://                        | auto       | [![Build Status][ci-develop-badge]][ci] |
```

Comment:
I think we can safely add the expected URLs here: `https://<appname>-develop.renuoapp.ch`